test/tests/functional/pbs_hook_execjob_end.py

```diff
@@ -213,3 +220,39 @@ def test_execjob_end_non_blocking_multi_node(self):
                 "execjob_end hook executed at: %s,"
                 "exechost_periodic at: %s and execjob_end hook ended at: %s" %
                 (date_time1, date_time2, date_time3))
+
+    def test_execjob_end_delete_request(self):
+        """
+        Test to make sure execjob_end hook is running
+        after job force deletion request(IS_DISCARD_JOB) when
+        mom is unblocked.
+        """
+        hook_name = "execjob_end_logmsg6"
+        self.server.create_import_hook(hook_name, self.attr, self.hook_body)
+        if len(self.moms) > 1:
```

Comment:
I think the check should be greater than 1 but less than2 since you are only requesting for 2 chunks.